### PR TITLE
Collection sorting 175

### PR DESCRIPTION
**Pull request by mccalluc**
@afred? Finish the fix for #175.

### CODE REVIEW

File: app/models/pb_core.rb [hunk 1]
```diff
@@ -28,6 +28,16 @@ def asset_title
     @asset_title ||= xpath_optional('/*/pbcoreTitle[@titleType="Open Vault Title"]')
   end
 
+  def title
+    @title ||= [series_title, program_title, asset_title].select { |x| x }.join('; ')
```
Comment (afred) [inline app/models/pb_core.rb]: @mccalluc why is `.select{ |x| x }` necessary?

Comment (afred) [inline app/models/pb_core.rb]: @mccalluc ^ ah.. to weed out `nil`?

Comment (mccalluc) [inline app/models/pb_core.rb]: yep. Maybe there's a better idiom?

Comment (afred) [inline app/models/pb_core.rb]: maybe `.select { |x| !x.nil? }` ... or..  `.reject(&:nil?)` would be more explicit... but oh well. It's merged.

**Pull request merged**